axum-extra/src/lib.rs

```diff
@@ -67,6 +68,9 @@ pub mod extract;
 pub mod response;
 pub mod routing;
 
+#[cfg(feature = "json-lines")]
+pub mod json_lines;
```

Comment:
In `axum`, only the `Json` type is exported at the root, not the `json` module. Does this deviate from that because there's more types to expose?

Reply:
Exactly!